stages/04.configure-desktop-env/02.vnc-server/files/adi-xserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# kuiper2.0 - Embedded Linux for Analog Devices Products
+#
+# Copyright (c) 2024 Analog Devices, Inc.
+# Author: Larisa Radu <larisa.radu@analog.com>
+
+# Check if the system has a display output
+if dmesg | grep -q "\[drm\]"; then
+	# Remove dummy display
+	if [ -e /usr/share/X11/xorg.conf.d/xorg.conf ]; then
```

Comment:
You don't need to add IF condition around. Just [use " -f " option on "rm"](https://unix.stackexchange.com/questions/280067/have-rm-not-report-when-a-file-is-missing#:~:text=Use%20the%20%2Df%20option.,will%20silently%20ignore%20nonexistent%20file).